packages/opentelemetry-basic-tracer/src/Span.ts

```diff
@@ -16,55 +16,55 @@
 
 import * as types from '@opentelemetry/types';
 import { performance } from 'perf_hooks';
-import { SpanKind, SpanContext } from '@opentelemetry/types';
+import { ReadableSpan } from './export/ReadableSpan';
 
 /**
  * This class represents a span.
  */
-export class Span implements types.Span {
-  private readonly _spanContext: types.SpanContext;
+export class Span implements types.Span, ReadableSpan {
   private readonly _tracer: types.Tracer;
-  private readonly _parentId?: string;
-  private readonly _kind: types.SpanKind;
-  private readonly _attributes: types.Attributes = {};
-  private readonly _links: types.Link[] = [];
-  private readonly _events: types.Event[] = [];
-  private _status: types.Status = {
+  readonly spanContext: types.SpanContext;
```

Comment:
Could you add some comments that these are included to implement `ReadableSpan` for export purposes but are not intended to be written-to directly?

Reply:
Sure, done.